user/views.py

```diff
@@ -76,10 +76,73 @@ def post(self, request):
                 return JsonResponse({'MESSAGE' : 'INVALID_EMAIL_OR_PASSWORD'}, status=400)
 
             access_token = jwt.encode({'id' : signed_user.id}, SECRET_KEY, algorithm=ALGORITHM)
-
             return JsonResponse({'MESSAGE' : 'SUCCESS', 'ACCESS_TOKEN' : access_token}, status=200)
         except ValueError:
             return ValueErrorTypeChecking(data)
         except KeyError:
             return JsonResponse({'MESSAGE' : 'INVALID_KEY'}, status=400)
 
+
+class PasswordCheckView(View):
+    @SignInAuthorization
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            user_id      = data['user_id']
+            password     = data['password']
+            signed_user  = User.objects.get(id=user_id)
```

Comment:
`get()` 호출 시 matching되는 데이터가 없을 경우 어떤 에러가 발생하나요?
```suggestion
            signed_user  = User.objects.get(id=user_id)
```

Reply:
User matching query does not exist. 발생합니다.

햬당 경우 예외처리 반영했습니다.